lib/models/notices.js

```diff
@@ -54,6 +54,14 @@ function processCBNotice(ncr) {
             }
             n[attr.name] = attr.value;
             n[attr.name + '__type'] = attr.type;
+
+            if (attr.metadata) {
+                Object.keys(attr.metadata).forEach(function(md) {
+                    if (attr.metadata[md].value) {
+                        n[attr.name + '__metadata__' + md] = attr.metadata[md].value;
```

Comment:
Probably this is due to my lack of expertise on JavaScript :) but, how this works for metadata which value is a JSON object?

(If the explanation is complex we can do it by skype)